scheduling-manager/src/cucumber/resources/features/scheduling/replace_after_terminate.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Deploying a Daemon environment to a cluster with Replace After Terminate
+
+  Background:
+    Given a Daemon environment named "DaemonEnvironment":
+      | clusterArn        | TestCluster           |
+      | deploymentMethod  | ReplaceAfterTerminate |
+      | taskDefinitionArn | v1                    |
+
+  Scenario: Deploying a new environment for the first time to an empty cluster
+    Given a cluster named "TestCluster" with the following instances and tasks:
```

Comment:
is the name important here?

Reply:
Hm, I think this is a vestige of when it was important. Will remove.

(Would ideally like to make it optional, but looks like that's not particularly easy)